CONTRIBUTING.md

```diff
@@ -2,104 +2,53 @@
-
-## Styleguides
-
-### Git Commit Messages
-
-* Use the present tense ("Add feature" not "Added feature")
-* Use the imperative mood ("Move cursor to..." not "Moves cursor to...")
-* Limit the first line to 72 characters or less
-* Reference issues and pull requests liberally after the first line
-* Consider starting the commit message with an applicable emoji:
-    * :art: `:art:` when improving the format/structure of the code
-    * :racehorse: `:racehorse:` when improving performance
-    * :non-potable_water: `:non-potable_water:` when plugging memory leaks
-    * :memo: `:memo:` when writing docs
-    * :penguin: `:penguin:` when fixing something on Linux
-    * :apple: `:apple:` when fixing something on macOS
-    * :checkered_flag: `:checkered_flag:` when fixing something on Windows
-    * :bug: `:bug:` when fixing a bug
-    * :fire: `:fire:` when removing code or files
-    * :white_check_mark: `:white_check_mark:` when adding tests
-    * :lock: `:lock:` when dealing with security
-
 ### Local Setup
 
-Genesis Custom Blocks development requires Node.js and follows the WordPress coding standards for PHP and JavaScript. In order to get your development environment setup quickly, simply run the following commands after cloning the plugin from Github:
+Genesis Custom Blocks generally follows WordPress coding standards for PHP and JavaScript.
 
-#### Node
+There's a pre-commit hook and linting scripts like `npm run lint`.
 
-**Install Packages**
+**Install Dependencies**
 
 ```
-npm install
+composer install && npm install
 ```
 
-This command installs required Node packages locally and is required before running build setup.
-
 **Watch Changes**
 
 ```
 npm run dev
 ```
 
-While developing, it is a best practice to watch for changes. This command will build assets as modifications are made.
+#### PHPUnit
 
-#### Composer
+```
+WP_TESTS_DIR=/path/to/wordpress-develop/tests/phpunit composer test
+```
+
+If you've already installed the DB for the [WordPress Core unit tests](https://github.com/WordPress/wordpress-develop/tree/0228dd6a5d17aa42735fdff9b106afccb960311e/tests/phpunit), simply pass the path to [wordpress-develop/tests/phpunit/](https://github.com/WordPress/wordpress-develop/tree/0228dd6a5d17aa42735fdff9b106afccb960311e/tests/phpunit) as `WP_TESTS_DIR`, as shown above.
+
+Otherwise, clone [wordpress-develop](https://github.com/WordPress/wordpress-develop). Then, `cd` back to this plugin and do:
 
 ```
-composer install
+WP_TESTS_DIR=/path/to/wordpress-develop/tests/phpunit ./bin/install-wp-tests.sh <db name> <db user> <db password>
 ```
 
+Use whatever DB name you'd like, and substitute a DB user and password that works in your environment.
+
+### Release Procedure
+
+1. `checkout` locally whatever branch you want to release. It could be `develop`, or a release branch like `1.0`. 
+1. Do `gulp`, and you'll see a `genesis-custom-blocks.zip` file in this plugin directory.
```

Comment:
See one of the suggestions here: https://github.com/studiopress/genesis-custom-blocks-pro/pull/1#pullrequestreview-473055317

Could we move this to `package` directory?

Reply:
Sure, good idea

Reply:
It's moved in https://github.com/studiopress/genesis-custom-blocks/pull/18/commits/ac6f3218ae5550d9cd27500d83417a6b79cc23d8